app/repositories/MongoJourneyAnswersRepository.scala

```diff
@@ -156,4 +164,17 @@ class MongoJourneyAnswersRepository @Inject() (mongo: MongoComponent, clock: Clo
       Updates.setOnInsert("expireAt", expireAt)
     )
   }
+
+  private def handleUpdateExactlyOne(ctx: JourneyContext, result: Future[UpdateResult])(implicit
+      logger: Logger,
+      ec: ExecutionContext): ApiResultT[Unit] = {
+    val futResult: Future[Either[ServiceError, UpdateResult]] = result.map { r =>
+      if (r.getModifiedCount != 1) {
```

Comment:
Is it worth having two different loggings, one for modified count = 0 and separate one for > 1? Could be useful to then have different errors for each case (when we get to adding pager duty).

Reply:
This one will have two different logging depends on the situation: I log r.getModifiedCount inside the message to know what was the actual number of records updated.